enumerable_methods.rb

```diff
@@ -0,0 +1,117 @@
+# frozen_string_literal: true
+
+module Enumerable
+  def my_each
+    i = 0
+    while i < length
+      yield self[i] if block_given?
+      i += 1
+    end
+    self
+  end
```

Comment:
* [ ] my_each returns an Enumerator if no block is given 
* [ ] my_each_with_index returns an enumerator if no block is given